hypertrace-view-generator/helm/values.yaml

```diff
@@ -300,3 +193,29 @@ kafka-topic-creator:
     address: zookeeper:2181
   imagePullSecrets: []
   podAnnotations: {}
+
+
+  generator:
+    deploymentName: span-event-view-generator
```

Comment:
Will this be separate from the grouped view gens?

Reply:
This code needs to be removed. 
Also let me add a description about the change